react-with-graphql-tutorial.md

```diff
@@ -1686,7 +1687,7 @@ const Organization = ({
 };
 {{< /highlight >}}
 
-Now, it can be defined in the App component. At this point, note that the `id` and the `viewerHasStarred` information could have been destructured from the App's local state too. That's how you wouldn't need to pass this information in the handler but use it from the local state instead. However, since the Repository component knew about the information already, it is fine to pass the information in the handler. You can argue that it makes the handler more explicit. Furthermore, in case you end up with multiple repositories and thus Repository components eventually, the handler would have to be specific to determine the correct repository anyway.
+Now it can be defined in the App component. Note that the `id` and the `viewerHasStarred` information can be destructured from the App's local state, too. This is why you wouldn't need to pass this information in the handler, but use it from the local state instead. However, since the Repository component knew about the information already, it is fine to pass the information in the handler, which also handler more explicit. It's also good preparation for dealing with multiple repositories and repository components later, since the handler will need to be more specific in these cases.
```

Comment:
"which also handler more explicit"  to " which also makes the handler more explicit"